bot-components/GitHub_GraphQL.ml

```diff
@@ -237,3 +250,49 @@ module MergePullRequest =
     }
   }
 |}]
+
+module NewCheckRun =
+[%graphql
+{|
+  mutation newCheckRun($name: String!, $repoId: ID!, $headSha: String!,
+  $status: String!, $title: String!, $text: String!, $summary: String!,
+  $conclusion: String!) {
+    createCheckRun(
+      input: {
+        status:$status,
+        name:$name,
+        repositoryId:$repoId,
+        headSha:$headSha,
+        conclusion:$conclusion,
+        output:{
+          title:$title,
+          text:$text,
+          summary:$summary
+        }
```

Comment:
You can (and you shoud) add the `detailsUrl` argument so that we can provide a different details URL than the default. (Cf. https://docs.github.com/en/graphql/reference/input-objects#createcheckruninput.)